docs/weaviate/modules/usage-modules.md

```diff
@@ -0,0 +1,170 @@
+---
+title: Usage Module
+description: Add the usage module to collect and upload usage analytics data to Google Cloud Storage (GCS) or AWS S3. 
+---
+
+## Overview
```

Comment:
We haven't been using the `## Overview` and `## Introduction` headings for most guides and references pages. I will add that to the style guide (which will be our for review next week 🥳)